UXWColors.js

```diff
@@ -1,13 +1,344 @@
+      <br/>
+      <br/>
+
+      <br/>
+      <div className="font-bold font-2xl">
       <p>
-        Lucas ipsum dolor sit amet carnor needa chazrach rebo sebulba plagueis finis di veknoid ventress.
-        Maul lando vader sy hutt nomi. Gordin haruun bail jar. Fett mothma toydarian jerec greedo bothawui sesswenna artaru zuckuss.
-        Nassau cerea sern naboo. Tagge leia obi-wan thennqora antilles. Davin selkath jobal tavion metalorn organa muunilinst jettster.
-        jinn hutt ikrit vor. Bertroff kael hutt wharl oola dagobah bibble saleucami. Colton raioballo ansionian hapan billaba wampa organa.
-        Greedo argazdan jinn tof verpine bothan wedge.
+        Philosophy
       </p>
+      </div>
+      <br/>
+
+      <p>
+      Our palette consists of neutral, pleasing tones balanced by vivid
+      and eccentric colors. This color palette has been carefully chosen
+      to provide versatility and function while still providing boundless
+      room for expression.
+      </p>
+
+      <br/>
+      <br/>
+      <br/>
+
+      <div className="font-bold font-2xl">
+        <p>
+          Using our palette
+        </p>
+        </div>
+        <br/>
+
+        <p>
+        Though our palette is established, we encourage experimentation
+        through blending modes and gradients. Start with the main variant in
+        the left column, and supplement your designs with the alternate shades
+        on the right if you want to play around. You might be surprised, the
+        results could be spellbinding!
+        </p>
+
+        <br/>
+        <br/>
+        <br/>
+
+        <div className="font-bold font-2xl">
+          <p>
+            Primary colors
+          </p>
+          </div>
+          <br/>
+
+          <p>
+          The flexible and neutral Magician's Cloak, Acid, and Merlin's Beard lead the way, providing a well-balanced foundation.
+          </p>
+
+          <br/>
+          <br/>
+          <br/>
+
+          <div className="font-bold font-2xl">
+            <p>
+              Magician's Cloak
+            </p>
+          </div>
+            <br/>
+
+           <div className="grid grid-cols-2 mx-4">
+             <div className="bg-indigo-300 text-center">Main variant</div>
+             <div className="bg-indigo-100 text-center">Alternate shades</div>
+           </div>
+
+
+
+           <div className="grid grid-cols-2 mx-4">
+
+             <div className="bg-indigo-300 h-32">
+                <div className="box h-32 w-32 bg-indigo-900 m-4"></div>  
+             </div>
+
+             <div className="bg-indigo-100 h-32">
+               <div className="grid grid-cols-2 gap-4 m-4">
+                 <div className="box h-16 w-16 bg-blue-400 m-4"></div>
+                 <div className="box h-16 w-16 bg-blue-900 m-4"></div>
+               </div>
+             </div>
+
+           </div>
+
+           <br/>
+           <br/>
+           <br/>
+
+

```

Comment:
Are you using VSCode? From the editor: If you right click within the contents of the file itself, you can press `Format Document` to have your text editor resolve formatting

[Link to Stack Overflow Thread](https://stackoverflow.com/questions/29973357/how-do-you-format-code-in-visual-studio-code-vscode)